pkg/ruler/ruler_test.go

```diff
@@ -277,6 +277,8 @@ func TestSharding(t *testing.T) {
 		shardingStrategy string
 		shuffleShardSize int
 		setupRing        func(*ring.Desc)
+		enabledUsers     []string
```

Comment:
nit: not sure if intended but we're still using the Users suffix throughout this test.

Reply:
I would need to change entire test to replace that. Doesn't seem to be worth the effort. I've changed ruler.go and compactor.go though.